### ISSUE TITLE
module 'tensorflow' has no attribute 'AdamOptimizer'

### ISSUE THREAD

**Issue by jly36963** [tensorflow/tensorflow#33989]
## Docs URL
https://www.tensorflow.org/api_docs/python/tf/estimator/DNNRegressor

## Tensorflow Version (that I'm using);
2.0.0

## Issue
I'm trying to create an "estimator using an optimizer with a learning rate decay". The documentation says to use `tf.AdamOptimizer`, `tf.exponential_decay`, and `tf.get_global_step`. Tensorflow has none of those attributes. I've tried playing around with suggestions from stack overflow (ie -- tf.train.AdamOptimizer), but I can't figure out where any of those attributes are actually located.

example code
```
model= tf.estimator.DNNRegressor(
    feature_columns=feature_columns,
    hidden_units = [100, 100, 100],
    optimizer=lambda: tf.AdamOptimizer(
        learning_rate=tf.exponential_decay(
            learning_rate=0.1,
            global_step=tf.get_global_step(),
            decay_steps=10000,
            decay_rate=0.96
        )
    )
)
```

**Comment by dubesar** [comment]
Instead of using `tf.AdamOptimizer` or `tf.train.AdamOptimizer` use `tf.optimizers.Adam()`

I hope this helps and clears the issue
Ping me if you don't get the issue cleared

**Comment by jly36963** [comment]
Thank you! that worked.

For anyone else who had a similar problem:
* Adam -- tf.optimizers.Adam()
* exponential_decay -- tf.optimizers.schedules.ExponentialDecay()
* I don't think `get_global_step()` is in version 2

**Comment by ymodak** [comment]
Closing this issue since its resolved. Thanks!
